azkaban/remote.py

```diff
@@ -332,6 +332,48 @@ def cancel_execution(self, exec_id):
       self._logger.info('Execution %s cancelled.', exec_id)
     return res
 
+  def pause_execution(self, exec_id):
+    """Pause workflow execution.
+
+    :param exec_id: Execution ID.
+
```

Comment:
Could you update the docstring with the behavior when pausing an execution that is already paused? Similar below when resuming.